docs/cli-commands.md

```diff
@@ -264,10 +264,7 @@ by IRIs provided in a mapping file.
 dsp-tools id2iri xmlfile.xml mapping.json
 ```
 
-The following options are available:
-
-- `--outfile` (optional, default: `id2iri_replaced_[timestamp].xml`): path to the output file
-- `-v` | `--verbose` (optional): print more information about the progress to the console
+The output file is written to `[original name]_replaced_[timestamp].xml`.
```

Comment:
both flags are not necessary:
 - outfile ist not necessary to be configurable. Just write the file with a suffix and a timestamp, that's enough
 - there were no substantial things printed with the verbose flag